src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java

```diff
@@ -82,7 +75,7 @@
     private final boolean searchShards;
     private final boolean parallel;
     private final boolean readOnly;
-    private final WriteTransaction writeTransaction;
+    private final SProcService.WriteTransaction writeTransaction;
```

Comment:
Some weird refactoring. Should it be `WriteTransaction` with 
```
import de.zalando.sprocwrapper.SProcService.WriteTransaction;
```
?

Reply:
Looks like a side effect of *organize imports*. I'll take a look.

Reply:
Any updates on this?